sockeye/arguments.py

```diff
@@ -1085,6 +1085,22 @@ def add_training_args(params):
                               action="store_true",
                               help='In addition to keeping the last n params files, also keep params from checkpoint 0.')
 
+    
+    train_params.add_argument('--cache-last-best-params',
+                              type=int,
+                              default=4,
+                              help='Cache the last n best params files to a separate directory, use 0 or negative to disable. Default: %(default)s')
+
+    train_params.add_argument('--cache-strategy',
+                              type=str,
+                              default="best",
```

Comment:
Can we use the string constants defined here instead: https://github.com/awslabs/sockeye/blob/master/sockeye/constants.py#L431

Reply:
yes, will address